types/backbone/index.d.ts

```diff
@@ -121,8 +121,8 @@ declare namespace Backbone {
         [event: string]: EventHandler;
     }
 
-    export const Events: Events;
-    interface Events extends EventsMixin {}
+    const Events: Events;
+    type Events = EventsMixin;
```

Comment:
This bit I'm a bit unsure about. Would it be possible to leave it out?

With hindsight, `EventsMixin` was a mistake; I wasn't aware at the time that declaration merging was a thing. I've been planning for a while to go back and do the right thing.